tests/e2e/tmp-default-cpu-scaling-mode/00-assert.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 90
+---
+apiVersion: vm.neon.tech/v1
+kind: VirtualMachine
+metadata:
+  name: example
+# temporary added this check to validate that defaulting cpuScalingMode works as expected
```

Comment:
>Why is it temporary? To me it seems it should be around as long as the field exists.

I think original idea was to eventually we drop the field, when we will be sure that sys fs scaling works

Reply:
> not forgetting to delete it

The test would break - impossible to forget. 

I'm just saying explicitly marking it as temporary gives the impression that the test is "more temporary", than the field.

Anyway, up to you.

Reply:
kept it as is